app/models/spree/flow_io_order_decorator.rb

```diff
@@ -172,8 +169,27 @@ def flow_order_authorized?
       flow_data&.[]('authorization') ? true : false
     end
 
-    def flow_order_captured?
-      flow_data['capture'] ? true : false
+    def flow_io_captures
+      flow_data&.[]('captures')
+    end
+
+    def flow_io_captures_sum
+      captures_sum = 0
+      # flow_data&.[]('captures')&.select { |c| c['status'] == 'succeeded' }&.map { |c| c['amount'] }&.sum.to_d
```

Comment:
Please delete commented code 🙏

Reply:
Yeah, it is deleted in the next PR targeted onto this branch - https://github.com/mejuri-inc/flowcommerce_spree/pull/25/files#diff-e683f17a9e0ac64e1001e420f4937ec3760eb6717e1707acb41d51c9fa2d446eL178